bam/bam_test.go

```diff
@@ -198,23 +198,15 @@ func BenchmarkReadIndex(b *testing.B) {
 	if *findex == "" {
 		b.Skip("no index file specified")
 	}
-	b.StopTimer()
-	f, err := os.Open(*findex)
-	if err != nil {
-		b.Fatalf("Open failed: %v", err)
-	}
-
-	buf, err := ioutil.ReadAll(f)
+	buf, err := ioutil.ReadFile(*findex)
 	if err != nil {
 		b.Fatalf("Index read failed: %v", err)
 	}
-	f.Close()
+	rdr := bytes.NewReader(buf)
```

Comment:
Sorry, but can this be `r`. We only live for 3 lines, so we know what it is by then.